js/model/dinnerModel.js

```diff
@@ -1,14 +1,28 @@
 // DinnerModel Object constructor
 var DinnerModel = function () {
+  var _this = this;
```

Comment:
Is there any reason not to use "this" directly?

Reply:
I think that it didn't work because of variable scope (see line 12).

Reply:
I will test it later